themes/Gruntfile.js

```diff
@@ -0,0 +1,266 @@
+          dest: 'docs/_vendor/jquery/'
+        }, {
+          expand: true,
+          cwd: 'node_modules/prismjs',
+          src: ['prism.js', 'themes/prism-okaidia.css'],
+          dest: 'docs/_vendor/prismjs/'
+        }, {
+          expand: true,
+          cwd: 'node_modules/bootstrap',
+          src: ['dist/**'],
+          dest: 'docs/_vendor/bootstrap/'
+        }]
+      },
+      css: {
+        files: [{
+          expand: true,
+          cwd: 'dist',
+          src: [
+            '**/*.css',
+            '**/*.scss'
+          ],
+          dest: DOCS_DEST
+        }]
+      }
+    },
+    sass: {
+      options: {
+        implementation: sass,
+        outputStyle: 'expanded'
+      },
+      dist: {
+        src: [],
+        dest: ''
+      },
+      docs: {
+        options: {
+          outputStyle: 'compressed',
+        },
+        src: 'docs/_assets/scss/custom.scss',
+        dest: 'docs/_assets/css/custom.min.css'
+      }
+    },
+    postcss: {
+      options: {
+        processors: [
+          autoprefixer({ cascade: false })
+        ]
+      },
+      dist: {
+        src: [],
+        dest: ''
+      },
+      docs: {
+        src: 'docs/_assets/css/custom.min.css',
+        dest: 'docs/_assets/css/custom.min.css'
+      }
+    },
+    cssmin: {
+      options: {
+        level: {
+          1: {
+            specialComments: 'all'
+          }
+        }
+      },
+      dist: {
+        src: [],
+        dest: ''
+      }
+    },
+    htmllint: {
+      options: {
+        ignore: [
+          /Attribute “autocomplete” is only allowed when the input type is.*/
+        ]
+      },
+      src: [
+        'docs/**/*.html',
+        '!docs/**/bower_components/**/*.html',
+        '!docs/_vendor/**/*.html',
+        '!docs/2/**/*.html'
+      ]
+    },
+    connect: {
+      options: {
+        hostname: 'localhost',
+        livereload: 35729,
+        port: 3000,
+        open: true
+      },
+      base: {
+        options: {
+          base: 'docs'
+        }
+      }
+    },
+    watch: {
+      options: {
+        livereload: '<%= connect.options.livereload %>',
+        spawn: false
+      },
+      dev: {
+        files: [
+          'dist/*/_variables.scss',
+          'dist/*/_bootswatch.scss'
+        ],
+        tasks: 'build'
+      }
+    }
+  });
+
+  grunt.registerTask('build', 'build a regular theme from scss', theme => {
+    theme = theme ? theme : buildTheme;
+
+    const themeDir = path.join(DIST_DIR, '/', theme);
+    const isValidTheme = grunt.file.exists(path.join(themeDir, '/_variables.scss')) && grunt.file.exists(path.join(themeDir, '/_bootswatch.scss'));
+
+    // cancel the build (without failing) if this directory is not a valid theme
+    if (!isValidTheme) {
+      grunt.log.writeln(`${theme} theme does not exist!`);
+      return;
+    }
+
+    const concatSrc = path.join(BUILD_DIR, '/scss/build.scss');
+    const concatDest = path.join(themeDir, '/build.scss');
```

Comment:
Based on my understanding, the two following files are generated and can be removed from the repository.